pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+supported chains. The Testnet executions are subsidized by Gelato and free. Make sure to create your 1Balance account
+using the same wallet address that you used to create the Gelato task.
+
+Switch network to Polygon
+![](../../images/guides/gelato/switch-to-polygon.png)
+
+Deposit USDC
+![](../../images/guides/gelato/deposit-usdc.png)
+
+**Pyth Fees**
+These are the fees that will be paid out to the Pyth oracle. The Pyth fees are paid in the native token of the chain
+where the task is running. The method that updates the price is payable, the update transaction has to include in the
+`msg.value` the corresponding fee.
+
+The on-chain transaction executed via a web3 function gets routed through a proxy smart contract
+which is solely owned by the web3 function task creator. This proxy smart contract is known as the `dedicatedMsgSender`
+and will be deployed the first time a task is created for a given wallet address.
+
+The `dedicatedMsgSender` is also responsible for paying the Pyth Fees. You will need to transfer over the native tokens
+to the `dedicatedMsgSender` on every chain where you want to run the task using a standard transfer.
+The `dedicatedMsgSender` address can be found in the Gelato task details under `Target smart contract` as the
+`Your msg.sender` field
+
+![](../../images/guides/gelato/dedicated-msg-sender.png)
+
+#### 5. Check task execution
+
+Once your task has been successfully created, you can check the task execution details in the Gelato app.
+[Example Pyth Web3 Function on Mumbai](https://beta.app.gelato.network/task/0x2c64539daaeb86addd6f556b68251a2fdf0df5c09ec07b8a30a2fbf76f4fe907?chainId=80001)
+![](../../images/guides/gelato/task-execution.png)
+
+## Managing your Gelato Task
+
+#### Monitoring
+
+To ensure that your tasks are running as expected, you should set up monitoring. Gelato supports monitoring 1Balance and
+address balances as well as execution statuses for tasks. You can set up monitoring for your Gelato balance and
+the `dedicatedMsgSender` address on each chain where you are running tasks and receive notifications in either
+Telegram or Discord.
+
+See the following links for more information:
+[1Balance Alerts](https://docs.gelato.network/developer-services/1balance/1balance-alerts)
+[Balance Alerts](https://docs.gelato.network/developer-services/automate/execution-and-balance-alerts)
+
+#### Updating configuration
+
+To update the configuration of your task, such as adding/removing new priceFeeds or changing the deviation thresholds,
+you can simply update the `config.yaml` file in your gist. The task will automatically pick up the changes and use them.
+Note that the `config.yaml` includes a `configRefreshRateInSeconds` parameter which determines how often the task will
+fetch the latest configuration from the gist vs reading a locally cached version. This is to prevent the task from being
+rate-limited by GitHub.
```

Comment:
can they check the log or something to see when the config change is picked up?

Reply:
they can check storage. added more details and screenshot